scripts/globals/status.lua

```diff
@@ -2077,11 +2077,12 @@ dsp.MAX_SLOTID  = 15
 
 dsp.objType =
 {
-    PC   = 0x01,
-    NPC  = 0x02,
-    MOB  = 0x04,
-    PET  = 0x08,
-    SHIP = 0x10,
+    PC     = 0x01,
+    NPC    = 0x02,
+    MOB    = 0x04,
+    PET    = 0x08,
+    SHIP   = 0x10,
+	TRUST  = 0x20,
```

Comment:
fix space

Reply:
its a tab. editorconfig not used and editor not configured to use 4 spaces

Reply:
I fixed the tabbing